### DISCUSSION TITLE
Ad hoc filter plugin for Postgresql datasource on grafana

### DISCUSSION THREAD

**Discussion by learnwithjj** [grafana/grafana#65584] [Plugins]
Is there any possible plugin for ad hoc filter for postgresql. As of now , grafana only supports ad hoc filter for Prometheus, Loki, InfluxDB, and Elasticsearch data sources. It could be of great help for users to customize the data selection instead of adding 100 where clauses in the backend. 
Please help, thanks 
![image](https://user-images.githubusercontent.com/83998131/228743123-4b2477de-d10e-46a7-b5a0-1f084b70266d.png)
![image](https://user-images.githubusercontent.com/83998131/228743324-be625691-8044-4624-9175-7c37becaaf66.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82780. Feel free to continue the discussion around this there. Thank you!
